File: tools-manage/invocationlab-admin/src/main/java/org/javamaster/invocationlab/admin/model/erd/GroupGetVo.java
package org.javamaster.invocationlab.admin.model.erd;

import lombok.Data;

import java.io.Serializable;
import java.util.Date;

@Data
public class GroupGetVo implements Serializable {
    private String id;
    private String configJSON;
    private String projectJSON;
    private String projectName;
    private String description;
    private String type;
    private String tags;
    private Object revision;
    private String delFlag;
    private String creator;
    private Date createTime;
    private String updater;
    private Date updateTime;
}

File: tools-manage/invocationlab-admin-21/src/main/kotlin/org/javamaster/invocationlab/admin/model/erd/GroupGetVo.kt
package org.javamaster.invocationlab.admin.model.erd

import org.javamaster.invocationlab.admin.annos.AllOpen

import java.io.Serializable
import java.util.*


@AllOpen
class GroupGetVo : Serializable {
    var id: String? = null
    var configJSON: String? = null
    var projectJSON: String? = null
    var projectName: String? = null
    var description: String? = null
    var type: String? = null
    var tags: String? = null
    var revision: Any? = null
    var delFlag: String? = null
    var creator: String? = null
    var createTime: Date? = null
    var updater: String? = null
    var updateTime: Date? = null
}
